tasks/redhat.yml

```diff
@@ -3,65 +3,72 @@
   yum:
     name: epel-release
     state: present
-  become: true
   register: result
   until: result is successful
 
 - name: redhat | Installing Pre-Reqs
   yum:
     name: "{{ postgresql_redhat_pre_reqs }}"
     state: present
-  become: true
   register: result
   until: result is successful
 
 - name: redhat | Installing Development Tools
   yum:
     name: "@Development tools"
     state: present
-  become: true
   register: result
   until: result is successful
 
 - name: redhat | Installing Postgresql Python Modules
   pip:
     name: ["psycopg2"]
     state: present
-  become: true
   register: result
   until: result is successful
 
 - name: redhat | Adding Postgresql Repo
   yum:
     name: https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm
     state: present
-  become: true
   register: result
   until: result is successful
 
+- name: redhat | Create version for paths
+  set_fact:
+    _postgresql_version: "{{ postgresql_version | replace('.', '') }}"
+
+- name: redhat | Facts for paths/commands
+  set_fact:
+    _postgresql_init_log: "/var/lib/pgsql/{{ postgresql_version }}/initdb.log"
+    _postgresql_bin: "/usr/pgsql-{{ postgresql_version }}/bin"
+
+- name: readhat | initdb command
+  set_fact:
+    # FIXME for 11, probably 10: _postgresql_init_db: "/usr/pgsql-{{ postgresql_version }}/bin/postgresql-{{ _postgresql_version }}-setup initdb"
```

Comment:
This is one thing I found that didn't work with 10/11 — I will add this for later though.

I think I'll send you a PR with a Matrix for various PostgreSQL versions. If it's okay, I keep the fixme until then.

Reply:
For this one here, I'd also move this to the `set_facts` as well when we get to the newer versions.

Reply:
Done!